app.rb

```diff
@@ -0,0 +1,188 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './student'
+
+class Library
+  def initialize
+    @rentals = []
+    @books = []
+    @people = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts
+      puts 'Books list is empty, but you can add a book'
+    end
+    @books.each { |book| puts "Title: '#{book.title}', Author: #{book.author}" }
+  end
+
+  def list_people
+    puts 'People list is empty' if @people.empty?
+    @people.each do |person|
+      puts "[#{person.class.name}] ID: '#{person.id}', Name: #{person.name}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student(1) or a teacher(2)? [input the number]'
+    ans = gets.chomp.to_i
+
+    puts 'Please input a number 1 for Student or 2 for Teacher.' if ans != 1 || ans != 2
+    while [1, 2].include?(ans)
+      case ans
+      when 1
+        create_student
+        break
+      when 2
+        create_teacher
+        break
+      end
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    while age < 1
+      print 'Please enter a valid age: '
+      age = gets.chomp.to_i
+    end
+
+    print 'Name: '
+    name = gets.chomp.capitalize
+    while name == ''
+      print 'Please enter a name: '
+      name = gets.chomp.capitalize
+    end
+
+    print 'Has parent permission? (Y/N) '
+    permission = gets.chomp.upcase
+
+    # rubocop:disable Lint/UnreachableLoop
+    while permission
+      case permission
+      when 'Y'
+        permission = true
+      when 'N'
+        permission = false
+      else
+        print 'Please enter Y for Yes or N for No: '
+        permission = gets.chomp.upcase
+      end
+      break
+    end
+    # rubocop:enable Lint/UnreachableLoop
+
```

Comment:
- [ ] Please try to avoid disabling linters as the help enforce good coding style which is better for you in the long run 👍